### PR TITLE
C2C-97: Add a product variant CSV to set the products and their prices and costs

### PR DESCRIPTION
**Pull request by reagan-meant**

### CODE REVIEW

File: configuration/product_variant/coupon.csv [hunk 1]
```diff
@@ -0,0 +1,3 @@
+id,lst_price,name,categ_id/id,type
+__export__.product_product_157,-50,Coupon,product.coupon,service
+__export__.product_product_300,-60,Coupon,product.coupon,service
```
Comment (rbuisson) [inline configuration/product_variant/coupon.csv:3]: Why are there 2 lines here?
Is this intentional?

File: configuration/product_variant/products.csv [hunk 2]
```diff
@@ -0,0 +1,38 @@
+id,name,standard_price,lst_price,uom_id/id,uuid,product_variant_ids/categ_id/id,type,uom_po_id/id
```
Comment (rbuisson) [inline configuration/product_variant/products.csv:1]: Rename the headers as per the specs:
`product_variant_ids/categ_id/id`

File: configuration/product_variant/lab_tests.csv [hunk 3]
```diff
@@ -0,0 +1,80 @@
+id,product_variant_ids/uuid,name,product_variant_ids/categ_id/id,type,lst_price
```
Comment (rbuisson) [inline configuration/product_variant/lab_tests.csv:1]: Rename the headers as per the specs:
`uuid` and `categ_id/id`

File: configuration/product_variant/visit_types.csv [hunk 4]
```diff
@@ -0,0 +1,3 @@
+id,name,product_variant_ids/categ_id/id,type,lst_price
```
Comment (rbuisson) [inline configuration/product_variant/visit_types.csv:1]: `categ_id/id`

**Review by rbuisson — CHANGES_REQUESTED** [review]
We need to remove the `product_variants_ids/` prefix in order to avoid duplicates.

File: configuration/product_variant/coupon.csv (hunk 1 above)
Comment (reagan-meant) [reply configuration/product_variant/coupon.csv:3]: Removed...It second line was added accidentally

File: configuration/product_variant/visit_types.csv (hunk 4 above)
Comment (reagan-meant) [reply configuration/product_variant/visit_types.csv:1]: Done

File: configuration/product_variant/lab_tests.csv (hunk 3 above)
Comment (reagan-meant) [reply configuration/product_variant/lab_tests.csv:1]: Done

File: configuration/product_variant/products.csv (hunk 2 above)
Comment (reagan-meant) [reply configuration/product_variant/products.csv:1]: Done

**Pull request merged**